db.json

```
@@ -1,11 +1,13 @@
 {
     "posts": [
       { "id": "1", "title": "a title", "views": 100 },
-      { "id": "2", "title": "another title", "views": 200 }
+      { "id": "2", "title": "another title", "views": 200 },
+      { "id": "3", "title": "Cong chua bong  bong 123", "views": 200 }
     ],
     "comments": [
       { "id": "1", "text": "a comment about post 1", "postId": "1" },
-      { "id": "2", "text": "another comment about post 1", "postId": "1" }
+      { "id": "2", "text": "another comment about post 1", "postId": "1" },
+      { "id": "3", "text": "doan xau nhat hanh tinh", "postId": "1" }
```

Dây là tiêu chuẩn quốc tế bạn Trang nên sửa lại
```suggestion
      { "id": "3", "text": "doan handsome nhat hanh tinh", "postId": "1" }
```